gravitee-apim-console-webui/src/management/api/general/details/api-general-info.component.ts

```diff
@@ -107,9 +111,9 @@ export class ApiGeneralInfoComponent implements OnInit, OnDestroy {
 
     this.apiId = this.activatedRoute.snapshot.params.apiId;
 
-    combineLatest([this.apiService.get(this.apiId), this.categoryService.list()])
+    combineLatest([this.apiService.get(this.apiId), this.categoryService.list(), this.tagService.list()])
```

Comment:
is it really necessary to add it to the combineLatest ?
Couldn't we simply do something like 

```typescript
    // in the component.ts
    public shardingTags: Observable<Tag[]> = this.tagService.list();
```
```html 
    // in the component.html
    <mat-option *ngFor="let tag of shardingTags$ | async" [value]="tag.id" [disabled]="tag.disabled">
```

Reply:
discussed and we will not do it